nodes/index.adoc

```diff
@@ -19,13 +19,13 @@ Using the OpenShift CLI (`oc`) or the web console, you can perform the following
 
 The following components of a node are responsible for maintaining the running of pods and providing the Kubernetes runtime environment.
 
-* Container runtime:: The container runtime is responsible for running containers. Kubernetes offers several runtimes such as containerd, cri-o, rktlet, and Docker.
+* Container runtime: The container runtime is responsible for running containers. Kubernetes offers several runtimes such as containerd, cri-o, rktlet, and Docker.
```

Comment:
You could either remove the bullet and retain double colon
OR
Make the component names bold for highlight

something like this
https://docs.openshift.com/container-platform/4.15/installing/index.html